crud/object.go

```diff
@@ -4,10 +4,14 @@ import (
 	"github.com/vmihailenco/msgpack/v5"
 )
 
-// Object is an interface to describe object for CRUD methods.
-type Object interface {
-	EncodeMsgpack(enc *msgpack.Encoder)
-}
+// Object is an interface to describe object for CRUD methods. It can be any
+// type that msgpack can encode as a map.
+type Object = interface{}
+
+// Objects is a type to describe an array of object for CRUD methods. It can be
+// any type that msgpack can encode, but encoded data must be an array of
+// objects.
```

Comment:
Maybe leave a link for #365 or provide an example from there to clarify "why it is an `interface{}` and not an `[]interface{}`"

Reply:
Updated.